src/app/zap-templates/zcl/data-model/chip/application-launcher-cluster.xml

```diff
@@ -26,57 +26,45 @@ limitations under the License.
     <description>This cluster provides an interface for launching content on a media player device such as a TV or Speaker.</description>
 
     <attribute side="server" code="0x0000" define="APPLICATION_LAUNCHER_LIST" type="ARRAY" entryType="INT16U" length="254" writable="false" optional="false">application launcher list</attribute>
-    <attribute side="server" code="0x0001" define="APPLICATION_LAUNCHER_CURRENT_APP" type="ApplicationLauncherEndpoint" writable="true" optional="false">application launcher app</attribute>
+    <attribute side="server" code="0x0001" define="APPLICATION_LAUNCHER_CURRENT_APP" type="ApplicationEP" writable="true" optional="false">application launcher app</attribute>
 
-    <command source="client" code="0x00" name="LaunchApp" optional="false">
+    <command source="client" code="0x00" name="LaunchAppRequest" response="LauncherResponse" optional="false">
       <description>Upon receipt, this SHALL launch the specified app with optional data. The TV Device SHALL launch and bring to foreground the identified application in the command if the application is not already launched and in foreground. The TV Device SHALL update state attribute on the Application Basic cluster of the Endpoint corresponding to the launched application. This command returns a Launch Response.</description>
       <arg name="data" type="CHAR_STRING"/>
```

Comment:
The type of `data` does not match the spec.  Followup, please.

Reply:
Added: https://github.com/project-chip/connectedhomeip/issues/13214